openmc/cell.py

```diff
@@ -344,6 +353,11 @@ def distribcell_paths(self, distribcell_paths):
                                string_types)
         self._distribcell_paths = distribcell_paths
 
+    @num_instances.setter
+    def num_instances(self, num_instances):
```

Comment:
Haha I debated this with myself. I'm fine with the `Geometry` directly incrementing the "private" `_num_instances` property. That's why this is written in Python after all :-)